test/api/core/entrypoints/embedded/index.test.js

```diff
@@ -112,6 +136,11 @@ describe('lib/core/api/core/entrypoints/embedded/index', () => {
         error: sinon.spy()
       }
     });
+
+    HttpMock.prototype.init = sinon.stub().resolves(true);
```

Comment:
DRY: putting `this.init = sinon.stub().resolves(true);` in the FakeProtocol constructor allows you to get rid of these lines

Reply:
No because I want to know separately when I call one of the `init` function

Reply:
I don't understand: with my proposal, won't each class instance have its own init function?

Reply:
Since I am instantiating theses classes in the EmbeddedEntrypoint, I have to set the stub directly on the prototype if I want to be able to change the returned value (eg: `MqttMock.prototype.init = sinon.stub().resolves(false);`)